src/scripts/main.js

```diff
@@ -1,3 +1,246 @@
 'use strict';
 
-// write your code here
+let board;
+let score = 0;
+const rows = 4;
+const columns = 4;
+const start = document.querySelector('.start');
+const restart = document.querySelector('.restart');
+const messageStart = document.querySelector('.message__start');
+const messageWin = document.querySelector('.message-win');
+const messageLose = document.querySelector('.message-lose');
+
+restart.style.display = 'none';
+
+start.addEventListener('click', () => {
+  setTwo();
+  setTwo();
+  start.style.display = 'none';
+  messageStart.style.display = 'none';
+  restart.style.display = 'block';
+});
+
+restart.addEventListener('click', () => {
+  for (let i = 0; i < rows; i++) {
+    for (let j = 0; j < columns; j++) {
+      board[i][j] = 0;
+
+      const tile = document.getElementById(i.toString() + '-' + j.toString());
```

Comment:
Could you use smth like this? Fix it everywhere, please
```suggestion
      const tile = document.getElementById(`${i}-${j}`);
```